docs/tutorials/multimodal/multimodal_prediction/document_classification.md

```diff
@@ -0,0 +1,128 @@
+from autogluon.multimodal.utils.misc import path_expander
+
+DOC_PATH_COL = "doc_path"
+
+train_data[DOC_PATH_COL] = train_data[DOC_PATH_COL].apply(lambda ele: path_expander(ele, base_folder=download_dir))
+test_data[DOC_PATH_COL] = test_data[DOC_PATH_COL].apply(lambda ele: path_expander(ele, base_folder=download_dir))
+print(test_data.head())
+```
+
+Let's display one of the document. 
+As you can, this is a budget document consisting of account number, account name, budgeted fund, expenditures, and etc.
+
+```{.python .input}
+from IPython.display import Image, display
+
+example_image = train_data.iloc[0][DOC_PATH_COL]
+pil_img = Image(filename=example_image, width=500)
+display(pil_img)
+```
+
+
+## Build a Scanned Document Classifier with AutoMM
+
+You can build a scanned document classifier with our MultiModalPredictor. 
+All you need to do is to create a predictor and fit it with the above training dataset. 
+Under the hood, AutoMM will automatically recognize handwritten or typed text, and make use of the recognized text, layout information, as well as the visual features for document classification. 
+Model customization is also quite simple, you can specify the underline foundation model using the `model.document_transformer.checkpoint_name` hyperparameter and AutoMM support document foundation models such as [layoutlmv3](https://huggingface.co/microsoft/layoutlmv3-base), [layoutlmv2](https://huggingface.co/microsoft/layoutlmv2-base-uncased), [layoutlm-base](https://huggingface.co/microsoft/layoutlm-base-uncased), [layoutxlm](https://huggingface.co/docs/transformers/model_doc/layoutxlm), etc., 
+as well as pure text models like [bert](https://huggingface.co/bert-base-uncased), [deberta](https://huggingface.co/microsoft/deberta-v3-base), just to name a few.
+
+Here, `label` is the name of the column that contains the target variable to predict, e.g., it is “label” in our example. 
+We set the training time limit to 180 seconds for demonstration purposes.
+
+```{.python .input}
+from autogluon.multimodal import MultiModalPredictor
+
+predictor = MultiModalPredictor(label="label")
+predictor.fit(
+    train_data=train_data,
+    hyperparameters={"model.document_transformer.checkpoint_name":"microsoft/layoutlmv2-base-uncased"},
+    time_limit=180,
+)
+```
+
+## Evaluate on Test Dataset
+
+You can evaluate the classifier on the test dataset to see how it performs:
+
+```{.python .input}
+scores = predictor.evaluate(test_data, metrics=["accuracy"])
+print('The test acc: %.3f' % scores["accuracy"])
+```
+
+## Predict on a New Document
+
+Given an example document, let’s visualize it first,
+```{.python .input}
+doc_path = test_data.iloc[1][DOC_PATH_COL]
+from IPython.display import Image, display
+pil_img = Image(filename=doc_path, width=500)
+display(pil_img)
+```
+
+We can easily use the final model to predict the label,
+```{.python .input}
+predictions = predictor.predict({DOC_PATH_COL: [doc_path]})
+print(predictions)
+```
```

Comment:
Do we want to say something like "Our prediction `0` is correct, given this is a budget document."

Reply:
Good point, will add it!